samples/hello-world/quickstart.yaml

```
@@ -1,80 +1,62 @@
-kind: Pod
 apiVersion: v1
-metadata:
-  name: foo-app
-  labels:
-    app: foo
-spec:
-  containers:
-  - name: foo-app
-    image: higress-registry.cn-hangzhou.cr.aliyuncs.com/higress/http-echo:0.2.3
-    args:
-    - "-text=foo"
----
 kind: Service
-apiVersion: v1
 metadata:
-  name: foo-service
-spec:
-  selector:
-    app: foo
-  ports:
-  # Default port used by the image
-  - port: 5678
----
-kind: Pod
-apiVersion: v1
-metadata:
-  name: bar-app
+  name: backend
   labels:
-    app: bar
+    app: backend
+    service: backend
 spec:
-  containers:
-  - name: bar-app
-    image: higress-registry.cn-hangzhou.cr.aliyuncs.com/higress/http-echo:0.2.3
-    args:
-    - "-text=bar"
----
-kind: Service
-apiVersion: v1
-metadata:
-  name: bar-service
-spec:
-  selector:
-    app: bar
   ports:
-  # Default port used by the image
-  - port: 5678
+    - name: http
+      port: 3000
+      targetPort: 3000
+  selector:
+    app: backend
 ---
-apiVersion: networking.k8s.io/v1
-kind: Ingress
+apiVersion: apps/v1
+kind: Deployment
 metadata:
-  name: foo
+  name: backend
 spec:
-  rules:
-  - http:
-      paths:
-      - pathType: Prefix
-        path: "/foo"
-        backend:
-          service:
-            name: foo-service
-            port:
-              number: 5678
+  replicas: 1
+  selector:
+    matchLabels:
+      app: backend
+      version: v1
+  template:
+    metadata:
+      labels:
+        app: backend
+        version: v1
+    spec:
+      containers:
+        - image: gcr.io/k8s-staging-ingressconformance/echoserver:v20221109-7ee2f3e
```

```suggestion
        - image: higress-registry.cn-hangzhou.cr.aliyuncs.com/higress/echoserver:v20221109-7ee2f3e
```
I upload this image to higress repo, so it is friendly to Chinese developers.